vuepress/docs/next/tutorials/consume/mt-redis.md

```diff
@@ -9,9 +9,11 @@ This tutorial describes the configuration process to add Redis for cache managem
 For more information, see [Entando Multitenancy].
 
```

Comment:
Add this: 
Checkout this project that installs a [Redis cluster with Sentinel using Helm](https://github.com/entando-ps/redis-sentinel).